app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPDescriptionRow.kt

```diff
@@ -0,0 +1,75 @@
+package com.twain.interprep.presentation.ui.components.generic
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import com.twain.interprep.R
+import com.twain.interprep.presentation.ui.theme.BackgroundDarkPurple
+
+@Composable
+fun IPDescriptionRow(modifier: Modifier = Modifier, title: String, body: String, icon: Int) {
+    Box(modifier = Modifier.fillMaxWidth()) {
+        Row() {
+            Icon(
+                painterResource(id = icon),
+                contentDescription = "Collection_bookmark",
+                modifier = Modifier
+                    .padding(dimensionResource(id = R.dimen.dimension_5dp))
+                    .size(dimensionResource(id = R.dimen.dimension_28dp)),
+                tint = BackgroundDarkPurple
+            )
+            Spacer(modifier = Modifier.width(dimensionResource(id = R.dimen.dimension_10dp)))
+            Column() {
+
+                Text(
+                    text = title,
+                    fontWeight = FontWeight.Bold,
+                    style = MaterialTheme.typography.bodyLarge
+                )
+                Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.dimension_10dp)))
+                Text(text = body, style = MaterialTheme.typography.bodyMedium)
+            }
+        }
+    }
+}
+@Preview
+@Composable
+fun IPIntroductionRow1Preview() {
+    val title = "Manage scheduled job interviews"
+    val description = "Effortlessly organize your interview details from a personalized dashboard"
+    IPDescriptionRow(title = title, body = description, icon = R.drawable.ic_dashboard)
```

Comment:
All 3 icons are already present in drawable as vector xmls. Please reuse those.

Reply:
Fixed it.